<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBoxDialog</class>
 <widget class="QDialog" name="AddBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додати бокс</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>28</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWidht">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHeight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLenght">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPosition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Позиція</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWidht">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHeight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Висота</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLenght">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Довжина</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>28</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosZ">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>28</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPosX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbPosY">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPosZ">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLenghtPoints">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Довжині</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWidhtPoints">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbKPO">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість точок по:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWidhtPoints">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширині</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLenghtPoints">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbHeightPoints">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Висоті</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHeightPoints">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbAngel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Кут</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAngle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBoxDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBoxDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
